From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

The example executables depend on certain example files being present.
Disable them.

diff -urN tomland-1.3.3.0/tomland.cabal tomland-1.3.3.0-r1/tomland.cabal
--- tomland-1.3.3.0/tomland.cabal	2021-10-31 05:34:34.387060464 -0600
+++ tomland-1.3.3.0-r1/tomland.cabal	2021-10-31 05:37:23.470076749 -0600
@@ -131,9 +131,7 @@
 
 executable readme
   import:              common-options
-  -- doesn't work on windows for unknown reasons
-  if os(windows)
-    buildable: False
+  buildable: False
   main-is:             README.lhs
   build-depends:       tomland
                      , text
@@ -144,9 +142,7 @@
 
 executable play-tomland
   import:              common-options
-  -- We are using DerivingVia that works only with > 8.6
-  if impl(ghc < 8.6)
-    buildable: False
+  buildable: False
   main-is:             Main.hs
   build-depends:       tomland
                      , bytestring
